src/main/java/eu/dissco/backend/controller/MasJobRecordController.java

```diff
@@ -45,21 +47,24 @@ public ResponseEntity<JsonApiListResponseWrapper> getMasJobRecordsForCreator(
       @PathVariable("creatorId") String creatorId,
       @RequestParam(defaultValue = DEFAULT_PAGE_NUM) int pageNumber,
       @RequestParam(defaultValue = DEFAULT_PAGE_SIZE) int pageSize,
-      @RequestParam(required = false) AnnotationState state,
+      @RequestParam(required = false) MasJobState state,
       HttpServletRequest request) {
     return ResponseEntity.ok().body(
-        service.getMasJobRecordsByCreator(creatorId, getPath(request), pageNumber, pageSize,
+        service.getMasJobRecordsByMasId(creatorId, getPath(request), pageNumber, pageSize,
             state));
 
   }
 
-  @GetMapping(value = "/{creatorIdPrefix}/{creatorIdSuffix}/{masJobId}/running")
+  @GetMapping(value = "/{masIdPrefix}/{masIdSuffix}/{jobIdPrefix}/{jobIdSuffix}/running")
```

Comment:
This won't look nice (20.1000.10/XXX-XXX-XXX/20.1000.10/YYY-YYY-YYY) but don't think we can do much about that

Reply:
I know :( 

Eventually, we will want to have MAS profiles for authorization, so that will reduce the endpoint complexity.